src/System.Composition.TypedParts/src/System/Composition/TypedParts/Discovery/DiscoveredPart.cs

```diff
@@ -236,10 +235,10 @@ private sealed class ParameterInfoComparer : IEqualityComparer<ParameterInfo>
         {
             public static readonly ParameterInfoComparer Instance = new ParameterInfoComparer();
 
-            public int GetHashCode(ParameterInfo obj)
-            {
-                return HashHelpers.Combine(obj.Position.GetHashCode(),  obj.Member.GetHashCode());
-            }
+            public int GetHashCode(ParameterInfo obj) => System.HashCode.Combine(
```

Comment:
No idea how to get this to compile locally, irregardless of what happens with the CI build that's a problem (as it prevents other developers from building locally).

Reply:
It looks like System.Composition.TypedParts only targets netstandard, not netcore and HashCode isn't in netstandard (yet). You probably just need to revert this one for now.